Integrations/SecBI/SecBI.py

```diff
@@ -0,0 +1,197 @@
+        """
+        params = get_dict_without_none(host=host, fields=SECBI_INCIDENT_FIELDS)
+        try:
+            raw_incident = self._http_request(method='GET', url_suffix=SECBI_API_ENDPOINT_INCIDENT, params=params)
+            secbi_incident = SecBIIncident(raw_incident)
+            return vars(secbi_incident)
+        except Exception as e:
+            LOG("Error when fetching Incident by host '{}' from SecBI: {}".format(host, e.args[0]))
+            LOG.print_log()
+            raise
+
+    def test_module(self, api_url, api_key):
+        """
+        Performs basic get request to test SecBI system
+        """
+        try:
+            self._http_request(method='GET', url_suffix=SECBI_API_ENDPOINT_STATUS)
+        except Exception as e:
+            return_error("Failed sanity test of SecBI with SECBI_API_URL '{}', and SECBI_API_KEY '{}': {}".format(api_url, api_key, e.args[0]))
+
+
+''' COMMANDS MANAGER / SWITCH PANEL '''
+
+
+def get_human_readable_incidents_list(incidents_list):
+    """
+    Format the Incident ID list to be of the format that ``tableToMarkdown`` accepts
+    :param incidents_list: the list of Incident IDs
+    :return: The result of ``tableToMarkdown`` on the supplied Incidents list
+    """
+    formatted = [dict(id=s) for s in incidents_list]
+    return tableToMarkdown('List of SecBI Incidents', formatted)
+
+
+def main():
+    LOG('Command being called is %s' % (demisto.command()))
+
+    params = demisto.params()
+    args = demisto.args()
+
+    SHOULD_VALIDATE_CERT = not params.get('insecure', False)
+    PROXY = params.get('proxy')
+
+    API_URL = params.get('API_URL')
+    API_KEY = params.get('API_KEY')
+
+    # Headers to be sent in requests
+    HEADERS = {
+        'secbi_api_key': API_KEY,
+        'Content-Type': 'application/json',
+        'Accept': 'application/json'
+    }
+    OK_CODES = (200, 204)
+
+    try:
+        client = SecBIClient(base_url=API_URL,
+                             verify=SHOULD_VALIDATE_CERT,
+                             proxy=PROXY,
+                             ok_codes=OK_CODES,
+                             headers=HEADERS)
+
+        if demisto.command() == 'test-module':
+            # This is the call made when pressing the integration test button.
+            client.test_module(API_URL, API_KEY)
+            demisto.results('ok')
+        elif demisto.command() == 'secbi-get-incidents-list':
+            query = args.get('query', None)
+            limit = args.get('limit', 100)
+            incidents_list = client.secbi_get_incidents_list(query, limit)
+
+            human_readable = get_human_readable_incidents_list(incidents_list)
+            entry_context = {
+                'SecBI.IncidentsList': incidents_list
+            }
+            return_outputs(human_readable, entry_context, incidents_list)
+        elif demisto.command() == 'secbi-get-incident':
```

Comment:
Please wrap the content of the condition in a command function which returns the outputs, and call it from here with `return_outputs`
Look [here](https://github.com/demisto/content/blob/master/Templates/Integrations/Authentication/Authentication.py#L447) for an example